spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/repository/query/AbstractNeo4jQuery.java

```diff
@@ -56,31 +65,29 @@ public final Object execute(Object[] parameters) {
 
 		Neo4jParameterAccessor parameterAccessor = getParameterAccessor(parameters);
 		ResultProcessor resultProcessor = queryMethod.getResultProcessor().withDynamicProjection(parameterAccessor);
-		return resultProcessor.processResult(new Neo4jQueryExecution.DefaultQueryExecution(neo4jOperations)
-				.execute(prepareQuery(resultProcessor, parameterAccessor), queryMethod.isCollectionLikeQuery()),
-			OptionalUnwrappingConverter.INSTANCE);
-	}
 
-	protected abstract PreparedQuery prepareQuery(ResultProcessor resultProcessor,
-		Neo4jParameterAccessor parameterAccessor);
+		PreparedQuery<?> preparedQuery = prepareQuery(resultProcessor.getReturnedType().getReturnedType(),
+			getInputProperties(resultProcessor), parameterAccessor, null, getMappingFunction(resultProcessor));
 
-	/**
-	 * @return True if the query should get a count projection applied.
-	 */
-	protected abstract boolean isCountQuery();
```

Comment:
Why is this removal not also part of the `AbstractReactiveNeo4jQuery`? Of course including all its derived classes.

Reply:
Done.